.github/workflows/push-deploy.yml

```diff
@@ -9,6 +9,8 @@ jobs:
     runs-on: ubuntu-latest
     steps:
     - uses: actions/checkout@master
+    - name: Build
+      run: composer install --prefer-dist --no-dev --optimize-autoloader
```

Comment:
I don't think that composer is available here. Probably needs https://github.com/pxgamer/composer-action.

For testing, can we run this on each commit to push it to trunk? Or on [schedule](https://help.github.com/en/actions/automating-your-workflow-with-github-actions/workflow-syntax-for-github-actions#onschedule)?

Reply:
Good idea. How can we test this without it committing to wp.org?

Reply:
If it only commits to trunk we should be okay I think.

Reply:
I have made the changes. I did a small test with https://github.com/wearerequired/wp-team-list/pull/63/checks?check_run_id=320778019 to see if the composer action works. The 10up action only works with tags and I could not see a way to test it.

I don't see any reason why it should not work.